src/shared/check_outdated_data.c

```diff
@@ -0,0 +1,29 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <time.h>
+
+int Run(int maxTimeDiff, char *aavild_data) {
+
+    time_t cur_time; 
+    //char * aavild_data = "20211122080000";
+    struct tm timeptr;
+
+    cur_time = time(NULL);
+    printf("Current timestamp = %ld\n", cur_time);
+
+    strptime(aavild_data, "%Y%m%d%H%M%S", &timeptr);
+    time_t aav_time = mktime(&timeptr);
+    printf("Aavild timestamp = %ld\n", aav_time);
```

Comment:
why are we printing here? and stop writing names inside the code like this

Reply:
just in order for me to see

Reply:
but i will just make it a comment